custom_components/rte_ecowatt/__init__.py

```diff
@@ -325,6 +325,7 @@ def _level2string(self, level):
         if self.state == STATE_ON and level == 3:
             return "Coupure d'électricité en cours"
         return {
+            0: "Situation normal - Electricité faible émission de CO2",
```

Comment:
Thank you for this change ! However, I believe it missed an "e": "Situation normale" (just like the line below). Same thing on line 381, of course.

Reply:
good catch!